components/x-article-save-button/src/ArticleSaveButton.jsx

```diff
@@ -41,7 +41,7 @@ export const ArticleSaveButton = (props) => {
 				aria-pressed={props.saved}
 			>
 				<span className="x-article-save-button__icon" />
-				{props.saved ? 'Saved' : 'Save'}
+				{!props.iconOnly ? (props.saved ? 'Saved' : 'Save') : ''}
```

Comment:
```suggestion
				{props.iconOnly ? '' : (props.saved ? 'Saved' : 'Save')}
```

_nit:_ might be a bit clearer if we invert the logic here and avoid having the logical not operator against `!props.iconOnly`?